ff/serving/runner/copy.go

```diff
@@ -148,3 +143,71 @@ func (m *CopyCompletionWatcher) String() string {
 	}
 	return "Job completed succesfully."
 }
+
+type MaterializedChunkRunnerConfig struct {
+	OnlineType     provider.Type
+	OfflineType    provider.Type
+	OnlineConfig   provider.SerializedConfig
+	OfflineConfig  provider.SerializedConfig
+	MaterializedID provider.MaterializationID
+	ResourceID     provider.ResourceID
+	ChunkSize      int64
+	ChunkIdx       int64
+}
+
+func (m *MaterializedChunkRunnerConfig) Serialize() (Config, error) {
+	config, _ := json.Marshal(m)
```

Comment:
don't drop these errors even if you think they won't throw an error. Someone might add something to config one day and cause marshal to fail and it'll be much harder to catch. Since you don't expect an error you can make it panic instead of returning an error.

Reply:
redone